inst/sql/sql_server/export/observationperiod/cumulativeduration.sql

```diff
@@ -1,13 +1,13 @@
 select 'Length of observation' as series_name, 
-	ar1.stratum_1*30 as x_length_of_observation, 
+	ar1.stratum_1 as x_length_of_observation,
 	round(1.0*sum(ar2.count_value) / denom.count_value,5) as y_percent_persons
-from (select analysis_id, cast(stratum_1 as int) stratum_1 from @results_database_schema.achilles_results where analysis_id = 108 GROUP BY analysis_id, stratum_1) ar1
+from (select analysis_id, cast(stratum_1 as int) * 30 stratum_1 from @results_database_schema.achilles_results where analysis_id = 108 GROUP BY analysis_id, stratum_1) ar1
 inner join
 (
-	select analysis_id, cast(stratum_1 as int) stratum_1, count_value from @results_database_schema.achilles_results where analysis_id = 108 GROUP BY analysis_id, stratum_1, count_value
+	select analysis_id, cast(stratum_1 as int) * 30 stratum_1, count_value from @results_database_schema.achilles_results where analysis_id = 108 GROUP BY analysis_id, stratum_1, count_value
 ) ar2 on ar1.analysis_id = ar2.analysis_id and ar1.stratum_1 <= ar2.stratum_1,
 (
 	select count_value from @results_database_schema.achilles_results where analysis_id = 1
 ) denom
-group by 30*ar1.stratum_1, denom.count_value
```

Comment:
I have seen a similar issue in postgresql, could you test and confirm this is still an issue if instead of the subselect you change line 12 to swap the order of column name and the 30*, as shown below?  It would be a smaller change.

group by ar1.stratum_1*30, denom.count_value

Reply:
We've merge a change that resolves the issue I saw on postgres and changed the order as I suggested.  Please pull latest and see if that works on BigQuery

Reply:
I've checked latest master and it does not work on BigQuery. BigQuery can't recognize column when multiplication is used and throws an error
`SELECT list expression references ar1.stratum_1 which is neither grouped nor aggregated`

Reply:
Could you please this as an issue on the OHDSI/SqlRender repository to see if there is a recommended change they could make for BigQuery support?

Reply:
Do you really need the `30*` in the GROUP BY? I would think

```
GROUP BY ar1.stratum_1, denom.count_value
```

should be sufficient?

Reply:
I just tested the approach suggested by @schuemie  on both pdw and redshift and it worked.  
@ssuvorov-fls  Can you confirm this works on BigQuery?

Reply:
@AnthonyMolinaro Yes, it works on BigQuery

Reply:
Ok, for clarity then, we would NOT use this PR and instead remove the 30* from the group by clause and that will work on all environments?

Reply:
Agree. Should I create new pr?

Reply:
Yes, that would be great thanks.